src/gui/tracks/TrackLabelButton.cpp

```diff
@@ -60,8 +63,8 @@ TrackLabelButton::TrackLabelButton( TrackView * _tv, QWidget * _parent ) :
 	else
 	{
 		setFixedSize( 160, 29 );
-		m_renameLineEdit->move( 30, ( height() / 2 ) - ( m_renameLineEdit->sizeHint().height() / 2 ) );
-		m_renameLineEdit->setFixedWidth( width() - 33 );
+		m_renameLineEdit->move( 26, ( height() / 2  - m_renameLineEdit->sizeHint().height() / 2 ) + 1 );
```

Comment:
```suggestion
		m_renameLineEdit->move(26, (height() / 2  - m_renameLineEdit->sizeHint().height() / 2) + 1);
```

We now follow this style convention where we don't use spaces within brackets. For every line modified, you should change this.

Reply:
sure, no problem.